data-prepper-plugins/failures-common/src/main/java/org/opensearch/dataprepper/plugins/dlq/s3/S3DlqWriter.java

```diff
@@ -47,7 +47,7 @@ public class S3DlqWriter implements DlqWriter {
     static final String S3_DLQ_REQUEST_SIZE_BYTES = "dlqS3RequestSizeBytes";
     static final String DLQ_OBJECTS = "dlqObjects";
     private static final String KEY_NAME_FORMAT = "dlq-v%s-%s-%s-%s-%s.json";
-    private static final String FULL_KEY_FORMAT = "%s/%s";
+    private static final String FULL_KEY_FORMAT = "%s%s";
```

Comment:
Yea, I took back one of my comments when I realized that it would break the configurations which are most likely correct. If was breaking odd configurations, I'd feel differently.

Reply:
Thanks y'all. let's continue the discussion on the github issue rather than here.